src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -38,11 +39,17 @@
 	private static final Logger LOGGER = LoggerFactory.getLogger(DLNAMediaAudio.class);
 	private AudioProperties audioProperties = new AudioProperties();
 
+	public static final int BITRATE_DEFAULT = 8000;
+	public static final int NUMBEROFCHANNELS_DEFAULT = 2;
+	public static final int BITSPERSAMPLE_DEFAULT = 16;
+	public static final int AUDIODELAY_DEFAULT = 0;
+	public static final int SAMPLEFREQUENCY_DEFAULT = 48000;
+
 	/**
 	 * @deprecated Use standard getter and setter to access this variable.
 	 */
 	@Deprecated
-	public int bitsperSample;
+	public int bitsperSample; //TODO: (Nad) Initialize?
```

Comment:
@Sami32 I use ```//TODO: (Nad)``` as "bookmarks" in the code when I'm working on it, for things I need to go back and either check or change. I've configured a "filter" in Eclipse that makes these lines of code pop up as "tasks" that I can click and jump directly to that code. I remove all of those before I make a final commit (hopefully at least) 😉